README.md

```diff
@@ -35,20 +35,41 @@ This requires a free FMOD account.
 
 ### Windows
 
-- Download the "FMOD Engine" package for Windows.
+- Download the "FMOD Studio" and ""FMOD Engine" package for Windows.
```

Comment:
The FMOD Engine package contains both the core and the studio libs, so I'm not sure why this change is necessary

Reply:
original brainfart moment. I will have another look and write the definitive guide.